README.md

```diff
@@ -156,7 +156,17 @@ https://192.168.37.1:8443/console/
 
 #### Step 3.1, Add a mobile app
 
-In your project, click on the mobile tab located in the left hand navigation. Next click ```create app``` fill in some details and pick cordova as the type. Then click create. (note) you will need to accept the cert of the mobile server. You can do this by hitting https://localhost:3001 in your browser.
+In your project (the `localmcp` project), click on the mobile tab located in the left hand navigation. 
+
+##### Provision the Mobile Control Panel
+
+For the first time it is needed to provision the Mobile Control Panel, for that click on the `Provision Mobile Control Panel` button and pick the `mcp` item from the Service Catalog. Select your project and move on. Once finished click the `View Project` button which brings you back to your project's overview. 
+
+##### Creating a Mobile Application
+
+Next click ```Create Mobile App``` fill in some details and pick cordova as the type. Then click create.
+
+**NOTE:** _You will need to accept the cert of the mobile server. You can do this by hitting https://localhost:3001 in your browser._  It's also recommended to open the browsers developer tools for more infos, in case there are problems with the self-signed certificate, or other issues.
```

Comment:
Good to have this as it's an issue everyone will have. You won't even see the option to create an App if the cert isn't accepted